pkg/proc/bininfo.go

```diff
@@ -474,22 +566,26 @@ func (c *nilCloser) Close() error { return nil }
 // LoadFromData creates a new BinaryInfo object using the specified data.
 // This is used for debugging BinaryInfo, you should use LoadBinary instead.
 func (bi *BinaryInfo) LoadFromData(dwdata *dwarf.Data, debugFrameBytes, debugLineBytes, debugLocBytes []byte) {
```

Comment:
We should rename this to something like `LoadImageFromData` and update the comment above to reflect the intent of the new changes.